File: fintechplatform/src/main/java/com/fintechplatform/android/transactions/db/Transaction.java
package com.fintechplatform.android.transactions.db;

import com.fintechplatform.android.db.PlatformDB;
import com.raizlabs.android.dbflow.annotation.Column;
import com.raizlabs.android.dbflow.annotation.PrimaryKey;
import com.raizlabs.android.dbflow.annotation.Table;
import com.raizlabs.android.dbflow.structure.BaseModel;

@Table(database = PlatformDB.class)
public class Transaction extends BaseModel {
    @PrimaryKey
    public String id;
    @Column
    public String who;
    @Column
    public String what;
    @Column
    public String twhen;
    @Column
    public String amount;
    @Column
    public Long order;
    @Column
    public String status;
    @Column
    public String resultcode;
    @Column
    public String message;
    @Column
    public String error;
    @Column
    public String accountId;

    public String getAccountId() {
        return accountId;
    }

    public void setAccountId(String accountId) {
        this.accountId = accountId;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getWho() {
        return who;
    }

    public void setWho(String who) {
        this.who = who;
    }

    public String getWhat() {
        return what;
    }

    public void setWhat(String what) {
        this.what = what;
    }

    public String getTwhen() {
        return twhen;
    }

    public void setTwhen(String twhen) {
        this.twhen = twhen;
    }

    public String getAmount() {
        return amount;
    }

    public void setAmount(String amount) {
        this.amount = amount;
    }

    public Long getOrder() {
        return order;
    }

    public void setOrder(Long order) {
        this.order = order;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getResultcode() {
        return resultcode;
    }

    public void setResultcode(String resultcode) {
        this.resultcode = resultcode;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getError() {
        return error;
    }

    public void setError(String error) {
        this.error = error;
    }
}

File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/transactions/db/Transaction.kt
package com.fintechplatform.ui.transactions.db

import com.fintechplatform.ui.db.PlatformDB
import com.raizlabs.android.dbflow.annotation.Column
import com.raizlabs.android.dbflow.annotation.PrimaryKey
import com.raizlabs.android.dbflow.annotation.Table
import com.raizlabs.android.dbflow.structure.BaseModel

@Table(database = PlatformDB::class)
data class Transaction(@PrimaryKey var id: String? = null,
                       @Column var who: String? = null,
                       @Column var what: String? = null,
                       @Column var twhen: String? = null,
                       @Column var amount: String? = null,
                       @Column var order: Long? = null,
                       @Column var status: String? = null,
                       @Column var resultcode: String? = null,
                       @Column var message: String? = null,
                       @Column var error: String? = null,
                       @Column var accountId: String? = null) : BaseModel()